packages/date/src/index.js

```diff
@@ -2,8 +2,8 @@
  * External dependencies
  */
 import momentLib from 'moment';
-import 'moment-timezone/moment-timezone';
-import 'moment-timezone/moment-timezone-utils';
+//import 'moment-timezone/moment-timezone';
+//import 'moment-timezone/moment-timezone-utils';
```

Comment:
I didn't manage to rely on the modules for moment-timezone, I had an error that I didn't understand. It will require a deeper look.